### ISSUE TITLE
[Feature]: Make the auto interval available as built in millisecond variable 

### ISSUE THREAD

**Issue by torkelo** [grafana/grafana#5846]
Example: 

```
SELECT (count("value") / $interval_ms) FROM "cpu_value" WHERE "type_instance" = 'idle' AND value < 0.5 AND $timeFilter GROUP BY time($interval) fill(null)﻿
```

$interval = 1m
$interval_ms = 60000

**Comment by champtar** [comment]
closed by #7190 ?

**Comment by torkelo** [comment]
Yes
